engine/tester/config_parse.c

```diff
@@ -184,6 +187,122 @@ xmlNodeNext(xmlNodePtr node)
     return xmlNodeSkipExtra(node->next);
 }
 
+/**
+ * Get the number of leading whitespaces in the line.
+ *
+ * @param line      Line for which leading whitespaces should be found.
+ * @param new_line  Set this pointer to next new line character.
+ *
+ * @return Number of leading whitespaces in the line, or @c -1 if there are
+           only whitespaces.
+ */
+static int
+get_leading_space_num(const char *line, const char **new_line)
+{
+    int         spaces_cnt = 0;
+    const char *p = line;
+    const char *end;
+
+    end = strchr(line, '\n');
+    if (new_line != NULL)
+        *new_line = end;
+    if (end == NULL)
+        end = line + strlen(line);
+
```

Comment:
strspn(line, " ") could help to make implementation smaller

also I guess it will remove ctype.h requirement